.htaccess

```diff
@@ -156,14 +156,67 @@
 
 	# Anything in sapphire/en/<version>/contributing is redirected to trunk, the info in there is usually not version specific,
```

Comment:
I wonder if we can actually drop these given they are commented out already? Thoughts @chillu ?